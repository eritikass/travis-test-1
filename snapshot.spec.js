const v = {
    a: 12345,
    b: 'kala',
    x: 'ertyuiop',
};

it('snapshot', () => {
    expect(v).toMatchSnapshot();
});