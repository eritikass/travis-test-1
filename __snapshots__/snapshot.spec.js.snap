// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot 1`] = `
Object {
  "a": 12345,
  "b": "kala",
  "x": "ertyuiop",
}
`;
